Src/BootCamp.Chapter/Item.cs

```diff
@@ -3,15 +3,19 @@
     public class Item
     {
         private string _name;
+        public string Name
+        {
+            get => _name;
+        }
         public string GetName()
```

Comment:
Can remove these `Get` methods now we have a property

Reply:
Request help because confused here.  Cannot get rid of GetName().  Test will fail, because the instructor's test expect item.GetName() in https://github.com/csinn/CSharp-From-Zero-To-Hero/blob/Chapter3/Homework/1/Tests/BootCamp.Chapter.Tests/PlayerTests.cs.

If I remove it, it will fail to pass and not be accepted.  If I do not change it, you will want it changed.  Infinite loop.  Request guidance.

Here are screenshots of the Test code that uses the GetName() method:

![image](https://user-images.githubusercontent.com/59816879/76693108-2f304a00-661d-11ea-87b6-8c0167a677d7.png)

![image](https://user-images.githubusercontent.com/59816879/76693113-3eaf9300-661d-11ea-8792-46e5d26d8ba7.png)

Reply:
Yep it was an accident in the homework - the tests should be using properties, so `item.GetName()` becomes `item.Name`

Reply:
Thank you for the reply.  I thought it was testing us and figuring out how to use multiple methods.  Also, your reply answered a question that I had about GitHub.